src/decode/decode.spec.ts

```diff
@@ -13,52 +13,30 @@ function hexStringToBytes(hexString: string): Uint8Array {
 }
 
 describe('decode', () => {
-  it.each<{ bytes: string; expected: CborValue }>([
+  describe.each<{ bytes: string; expected: CborValue }>([
     // ###### SIMPLE ######
 
     { bytes: 'F4', expected: false }, // false
-    { bytes: 'D9D9F7F4', expected: false }, // self-described false
```

Comment:
The self-described test cases were all removed and the self-described tag is now added in the test itself. This removes a lot of repeated test cases.